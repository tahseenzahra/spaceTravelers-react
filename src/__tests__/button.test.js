import renderer from 'react-test-renderer';

import Button from '../UI/button';

describe('Button', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(<Button />);
  });

  it('should render button correctly', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});