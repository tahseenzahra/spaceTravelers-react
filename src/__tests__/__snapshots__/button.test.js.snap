// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render button correctly 1`] = `
<button
  type="button"
/>
`;
